src/modules/EnvironmentVariables/EnvironmentVariables/EnvironmentVariablesXAML/MainWindow.xaml.cs

```diff
@@ -31,17 +32,20 @@ public MainWindow()
             AppTitleTextBlock.Text = title;
 
             RegisterWindow();
+
+            WindowHelpers.BringToForeground(handle);
         }
 
         private static readonly DispatcherQueue _dispatcherQueue = DispatcherQueue.GetForCurrentThread();
         private static NativeMethods.WinProc newWndProc;
         private static IntPtr oldWndProc = IntPtr.Zero;
+        private static IntPtr handle = IntPtr.Zero;
```

Comment:
no need to make this a class field. You can obtain it locally and in ctor and just pass it to RegisterWindow()